vip-real-time-collaboration.php

```diff
@@ -60,3 +64,48 @@
 	// Fire action to indicate that the plugin has loaded.
 	do_action( 'vip_real_time_collaboration_loaded' );
 }, 10, 0 );
+
+/**
+ * Verify that we can initialize the VIP Real-Time Collaboration plugin.
+ *
+ * @global string $wp_version The WordPress version string.
+ *
+ * @return bool true if the plugin can load, false otherwise.
+ */
+function vip_real_time_collaboration_pre_init(): bool {
+	$php_version = phpversion();
+	if ( is_string( $php_version ) && version_compare( $php_version, '8.2', '<' ) ) {
+		add_action( 'admin_notices', function (): void {
+			wp_admin_notice(
+				__(
+					'The VIP Real-Time Collaboration plugin requires PHP 8.2+. The VIP Real-Time Collaboration plugin has been disabled.',
+					'vip_real_time_collaboration'
+				),
+				[ 'type' => 'error' ]
+			);
+		}, 10, 0 );
+		return false;
+	}
+
+	global $wp_version;
+
+	// Account for plugins overriding the $wp_version global, look at Gutenberg.php for reference.
+	/** @psalm-suppress MissingFile */
+	// This is a built-in WordPress file, so we can ignore the warning here.
+	require ABSPATH . WPINC . '/version.php';
```

Comment:
This matches how `gutenberg.php` does it. A little safer. Unless there's a reason we should `require`.
```suggestion
	include ABSPATH . WPINC . '/version.php';
```

Reply:
Require will halt the execution if its unable to get the version.php loaded, while include will continue on. Noticed in core that, they used require - https://github.com/search?q=repo%3AWordPress%2Fwordpress-develop%20ABSPATH%20.%20WPINC%20.%20%27%2Fversion.php%27%3B&type=code so I ended up switching it to that as well.

Happy to match it with Gutenberg, can re-visit this if we need to.